source/views/stoprint/print-jobs.js

```diff
@@ -103,20 +103,17 @@ class PrintJobsView extends React.PureComponent<Props> {
 				/>
 			)
 		}
-		if (this.props.loginState !== 'logged-in') {
-			return (
-				<StoPrintNoticeView
```

Comment:
And I removed this from testing... going to add it back.

Reply:
I shouldn't have been so quick to approve you! 😝